stock_serial_batch_receive/views/stock_move.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_stock_move_operations" model="ir.ui.view">
+        <field name="name">stock.move.operations.form</field>
+        <field name="model">stock.move</field>
+        <field name="inherit_id" ref="stock.view_stock_move_operations" />
+        <field name="arch" type="xml">
+            <xpath expr="//group/group" position="after">
+                <group>
+                    <field name="has_tracking" invisible="1" />
+                    <field name="picking_type_create_lots" invisible="1" />
+                </group>
+            </xpath>
+
+            <xpath expr="//button[@special='cancel']" position="after">
+                <button name="open_generate_serials_wizard" string="Generate serial numbers" type="object" />
```

Comment:
Can you please move this button before the tree view, below the field quantity done or on the right side group? If it's in footer, you would expect it to close the actual modal, but it's not what it does, and it doesn't feel right...

Moreover this button must be invisible if product_id.tracking != 'serial'

![image](https://user-images.githubusercontent.com/778828/63932180-fbae1980-ca56-11e9-977c-4d95f1dd1656.png)

Reply:
I thought it will be more convenient to have all the buttons together,
 updated